### PR TITLE
Video organization

### PR DESCRIPTION
**Pull request by ChuckBTaylor**
Not sure what happened. I thought I'd merged this

### CODE REVIEW

File: src/components/templates/SeriesListPage.tsx [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
+  const theme = useTheme()
+  const barColor = theme.palette.series.main
   const seriesListSubheaderText =
     'Series List - Risus velit lectus ut vitae sit nunc mauris mattis erat. Posuere adipiscing nisi lacinia lacinia facilisi venenatis aliquam.'
 
   return (
     <>
       <PageTitle
         title="Series"
+        titleUnderlineColor={barColor}
```
Comment (kmdunn5) [inline src/components/templates/SeriesListPage.tsx:16]: Why not just use `theme.pallete.series.main` here? Why set the variable?

File: src/theme.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
-import { createTheme } from '@mui/material'
+import { createTheme } from '@mui/material/styles'
 
-export const kurtTheme = createTheme({
+const theme = createTheme({
```
Comment (kmdunn5) [inline src/theme.tsx:3]: Thank you

File: src/theme.tsx [hunk 3]
```diff
@@ -46,43 +46,22 @@ export const kurtTheme = createTheme({
...
       },
     },
     MuiCard: {
       defaultProps: {
         elevation: 0,
-        sx: {
-          backgroundColor: '#212529',
-          overflow: 'visible',
-        },
+        sx: { backgroundColor: '#0C0D0D' },
```
Comment (kmdunn5) [inline src/theme.tsx:58]: Should we set any of these colors to variables somewhere so they are easier to reuse? And so we can look at the names and know what color it is quicker?

**Review by kmdunn5 — COMMENTED** [review]
My only change here would be to have some variables names for our colors. That will make it easier to communicate what color things need to be in the future. And prevents us from using wrong colors on accident.

File: src/theme.tsx (hunk 3 above)
Comment (ChuckBTaylor) [reply src/theme.tsx:58]: Makes sense, ya

File: src/components/templates/SeriesListPage.tsx (hunk 1 above)
Comment (ChuckBTaylor) [reply src/components/templates/SeriesListPage.tsx:16]: Dunno. Could have. Might have been thinking of doing something more.

File: src/components/molecules/contactForm/ContactForm.tsx [hunk 4]
```diff
@@ -1,60 +1,115 @@
...
-        : <>
-            {inputs.map((i) => <TextInput {...inputToFormProps[i]} onChange={(e: React.ChangeEvent<HTMLInputElement>) => updateFieldState(i, e.target.value)}/>)}
-            <Button type='submit' variant='contained' disabled={!Object.values(fieldStates).every(val => val !== '')}>Send</Button>
-          </>
-      }
-      
-    </Box>
+    <>
+      {!formSubmitted && (
+        <Grid
```
Comment (ChuckBTaylor) [inline src/components/molecules/contactForm/ContactForm.tsx:79]: Used `Grid` since it solved some of the formatting issues I was having.

**Review by kmdunn5 — APPROVED** [review]

**Pull request merged**